load_sample_data.sh

```diff
@@ -17,31 +17,40 @@ docker-compose exec -T "$WEB_SERVICE_NAME" /bin/bash -c "python3 manage.py shell
 
 # Migrate analytics
 docker-compose exec -T "$ANALYTICS_SERVICE_NAME" /bin/bash -c "PYTHONPATH=. pipenv run alembic upgrade head"
-# Copy table `image` from database to upstream database
-docker-compose exec -T "$UPSTREAM_DB_SERVICE_NAME" /bin/bash -c "PGPASSWORD=deploy pg_dump -s -t image -U deploy -d openledger -h db | psql -U deploy -d openledger"
+
 # Load content providers
 docker-compose exec -T "$DB_SERVICE_NAME" /bin/bash -c "psql -U deploy -d openledger <<-EOF
 	INSERT INTO content_provider (created_on, provider_identifier, provider_name, domain_name, filter_content) VALUES (now(), 'flickr', 'Flickr', 'https://www.flickr.com', false), (now(), 'behance', 'Behance', 'https://www.behance.net', false);
 	EOF"
-# Load sample data, including content providers
+
 docker-compose exec -T "$UPSTREAM_DB_SERVICE_NAME" /bin/bash -c "psql -U deploy -d openledger <<-EOF
-	ALTER TABLE image RENAME TO image_view;
-	ALTER TABLE image_view ADD COLUMN standardized_popularity double precision;
 	CREATE TABLE content_provider(provider_identifier varchar(50), provider_name varchar(250), created_on timestamp, domain_name varchar(500), filter_content boolean, notes text);
 	INSERT INTO content_provider (created_on, provider_identifier, provider_name, domain_name, filter_content) VALUES (now(), 'flickr', 'Flickr', 'https://www.flickr.com', false), (now(), 'behance', 'Behance', 'https://www.behance.net', false);
+	EOF"
+
+# Load sample data for images
+docker-compose exec -T "$UPSTREAM_DB_SERVICE_NAME" /bin/bash -c "PGPASSWORD=deploy pg_dump -s -t image -U deploy -d openledger -h db | psql -U deploy -d openledger"
+docker-compose exec -T "$UPSTREAM_DB_SERVICE_NAME" /bin/bash -c "psql -U deploy -d openledger <<-EOF
+	ALTER TABLE image RENAME TO image_view;
+	ALTER TABLE image_view ADD COLUMN standardized_popularity double precision;
```

Comment:
This file looks clearer ✨ The `sample_data.tsv` of images now also has an `ingestion_type` column so it should change too but that is probably for another PR, if possible.

Reply:
The [sample data for images](https://github.com/WordPress/openverse-api/blob/c27aeac9ab6ba51e65a44eb5a3d54fbb980f91a1/sample_data/sample_data.csv) has not been updated yet so we should make a new issue and PR for that.